.github/workflows/docs-check.yml

```diff
@@ -33,6 +33,15 @@ jobs:
         with:
           python-version: 3.9
 
+      - name: Run helm-docs
+        run: |
+          make helm-docs
```

Comment:
How about pushing a change to the PR to show this action will fail if the docs is out of date? (or perhaps you already confirmed this)

Reply:
Tested: 
![image](https://user-images.githubusercontent.com/3438324/204624690-46d8fc44-219a-46eb-b052-a9ab680aea2c.png)